src/me/mrCookieSlime/Slimefun/Lists/SlimefunItems.java

```diff
@@ -361,6 +361,7 @@
 	public static ItemStack GRIND_STONE = new CustomItem(Material.DISPENSER, "&bGrind Stone", 0, new String[] {"", "&a&oGrinds Items for more Efficiency"});
 	public static ItemStack ARMOR_FORGE = new CustomItem(Material.ANVIL, "&6Armor Forge", 0, new String[] {"", "&a&oGives you the Ability to create powerful Armor"});
 	public static ItemStack SMELTERY = new CustomItem(Material.FURNACE, "&6Smeltery", 0, new String[] {"", "&a&oActs as a high-temperature Furnace for Metals"});
+	public static ItemStack IGNITION_CHAMBER = new CustomItem(new ItemStack(Material.HOPPER), "&4Automatic Ignition Chamber", "&rAutomatically ignites the block under the Smelter,", "&ron which it's installed", "&e&oRequires Flint'n Steels to run");
```

Comment:
It's Smeltery, not Smelter.
Also, a little misleading as it doesn't ignite the Block. It just prevents it from going out.

"Prevents the Smeltery from using up fire."
"Requires Flint and Steel"
"Must be placed adjacent to the Smeltery's dispenser"